packages/connect/src/connector/specs/index.ts

```diff
@@ -1,5 +1,13 @@
+const mapNameToId: Map<string, string> = new Map<string, string>([
+  ["polkadot", "polkadot"],
+  ["ksmcc3", "kusama"],
+  ["rococo_v1_13", "rococo"],
+  ["westend2", "westend"],
+])
```

Comment:
Safer:

```suggestion
const mapNameToId: Record<WellKnownChains, string> = {
  [WellKnownChains.polkadot], "polkadot",
  [WellKnownChains.ksmcc3], "kusama",
  [[WellKnownChains.rococo_v1_13] "rococo",
  [WellKnownChains.westend2], "westend",
}
```

This is better and safer for 2 reasons: one the one hand it ensures that all the entries of `WellKnownChains` have an entry inside the Record, on the other when accessing `mapNameToId` we don't have to worry about possible `undefined` values.

Reply:
This is not needed anymore - since the naming of the files have altered